hummingbot/client/config/config_var.py

```diff
@@ -40,10 +40,10 @@ def required(self) -> bool:
         assert callable(self._required_if)
         return self._required_if()
 
-    def validate(self, value: str) -> bool:
+    def validate(self, value: str) -> str:
```

Comment:
The return type should be `Optional[str]`, since for successful validation it should return None.

Reply:
I have addressed this in this PR
https://github.com/CoinAlpha/hummingbot/pull/1615